vdgsa_backend/rental_viols/models.py

```diff
@@ -1,3 +1,266 @@
+from __future__ import annotations
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+class Bow(models.Model):
+    bow_num = models.AutoField(primary_key=True)
+    vdgsa_number = models.IntegerField(blank=True, null=True)
+    maker = models.CharField(max_length=50, blank=True, null=True)
+    size = models.CharField(max_length=9, blank=True, null=True)
+    state = models.CharField(max_length=10, blank=True, null=True)
+    value = models.DecimalField(max_digits=6, decimal_places=2, blank=True, null=True)
+    provenance = models.TextField(blank=True, null=True)
+    description = models.TextField(blank=True, null=True)
+    accession_date = models.DateField(blank=True, null=True)
+    notes = models.TextField(blank=True, null=True)
+    viol_num = models.ForeignKey(
```

Comment:
Suggest renaming `xxx_num` relationship fields to just `xxx`, e.g. `viol_num` -> `viol`.

Also question: should these be foreign key or one-to-one? If they're one-to-one, should we put them on the "Viol" model?